modules/ownKeys.js

```diff
@@ -0,0 +1,16 @@
+import objectKeys from 'object-keys'
+
+const reflectOwnKeysExist =
+  typeof Reflect === 'object' && typeof Reflect.ownKeys === 'function'
+
+const ownKeys = (object) => {
+  if (reflectOwnKeysExist) {
+    return Reflect.ownKeys(object)
+      .filter(key => Object.getOwnPropertyDescriptor(object, key).enumerable)
+  }
```

Comment:
This method will currently fall back to Object.keys even in browsers that support `Object.getOwnPropertySymbols` - can we add a third option here, that if `Object.getOwnPropertySymbols` is a function, we return both keys and symbols?